firmware/application/apps/ui_spectrum_painter.cpp

```diff
@@ -69,7 +69,7 @@ SpectrumPainterView::SpectrumPainterView(
         };
     };
 
-    tx_gain = 10;
+    tx_gain = portapack::transmitter_model.tx_gain();
```

Comment:
I think this is also partly due to the fact that the TransmitterModel doesn't set tx_gain in enable so you're getting whatever the value that was set if the ReceiverModel was enabled. I'll fix that in #1056.

Reply:
Another option here is to maybe to not use a local tx_gain at all and just use the transmitter_model. This call could be `transmitter_model.set_tx_gain(10);` instead.

Reply:
The behavior suggests that if transmitter_model.set_tx_gain() is never called after startup, then the default TX gain is 47. Perhaps 10 would be a better initial value system wide. Or maybe some folk would like that to be configurable.

Reply:
I have no idea where 47 came from, or 10 for that matter. There's a bug in transmitter_model where enable doesn't (didn't until #1056) actually call set_tx_gain on the radio in enable. So it's technically uninitialized which might explain some of the strangeness. @bernd-herzog might have a better explanation for the value 10.

Reply:
Also, 47 looks like the max value - or something is limiting it to 47 in the view.
There's also this from the TX View coloring:
```
#define POWER_THRESHOLD_HIGH 47
#define POWER_THRESHOLD_MED 38
#define POWER_THRESHOLD_LOW 17
```

So 10 is pretty low -- which might be a good idea. 47 as a default doesn't sound like a great idea.

Reply:
The reason I used 10 was because I needed to turn it down to 0 every time I tested the transmission to not blast my neighborhood with "nonsense rf". Going down to 0 from 10 is faster than from 47. I never noticed the default value did not work because I changed it anyway.

Change looks good for me. Thanks for the effort.

Reply:
HI @kallanreed , those 3 threshold power levels that you paste here ,  are  evaluated with (GAIN + AMP )
Then 47 is where start HIGH power  level , but the max is 47 +14 =61.  (Sol all that range is coloured as RED = HIGH (47...61)

Anyway , as a default for all transmitters I would suggest, GAIN =35 , AMP = 0 . (That means inside MID power colour level  ) 

Cheers

Reply:
Opened #1062 to set default to 10.

Reply:
Hi @aSmig ,  accidentally I just found the same bug than you today . 
Then , we need to correct that bug ,  but if we want to communize the initial GAIN set up of all Transmitters, I would suggest to use init GAIN of 35 (AMP = 0=   .  (using  rod telescopic antenna is really low range, and safe and respectful to our neighbor .)  I alread added my comments to the #1062 .

Reply:
Between this change, 1056 and 1062, this should be good now.

Reply:
Confirmed, testing the binary of  those 2 merged, #1056,  #1062 ,  
Now  all Transmitter App have the correct GAIN ,35, and Painter Spectrum does not have the previous bug .
Tomorrow's nightly will be ok !